### PR TITLE
Closes #1 Create initial index.html markup

### LINKED ISSUE #1: Create initial index.html markup

### PR DESCRIPTION
**Pull request by kisiohlova**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+</head>
+
+<body></body>
```
Comment (dafeys) [inline index.html]: Closing tag on the new line

**Review by dafeys — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+
+<html lang="en">
```
Comment (obniavko) [inline index.html:3]: Added language.

**Review by obniavko — APPROVED** [review]

**Review by fh0enix — APPROVED** [review]
Okey

**Review by dafeys — APPROVED** [review]
LGTM

**Review by DmytroStoliaruk — APPROVED** [review]

**Review by iryna-borniak — APPROVED** [review]
🔥

**Pull request merged**